cfgov/unprocessed/apps/analytics-gtm/js/util/analytics-util.js

```diff
@@ -25,6 +25,33 @@ function addEventListenerToElem( elem, event, callback ) {
   }
 }
 
+/**
+ * Query a selector and remove listeners from returned elements.
+ * @param {string} selector - A dom selector.
+ * @param {string} event - An event string, probably a "MouseEvent."
+ * @param {Function} callback - The event handler.
+ */
+function removeEventListenerFromSelector( selector, event, callback ) {
+  const elems = document.querySelectorAll( selector );
+  for ( let i = 0, len = elems.length; i < len; i++ ) {
+    removeEventListenerFromElem( elems[i], event, callback );
+  }
+}
+
+/**
+ * Check if an element exists on the page, and if it does, remove listeners.
+ * @param {[type]}   elem     [description]
+ * @param {[type]}   event    [description]
```

Comment:
event and callback can be the same descriptions as those set in `removeEventListenerFromSelector`.